src/volt/components/VoteControls/index.js

```diff
@@ -224,7 +222,7 @@ class VoteControls extends Component {
     const contractInterface = new utils.Interface(abi);
 
     return contractInterface.functions.castBallot.encode([
-      parseInt(balanceCardId),
+      parseInt(balanceCardId, 10),
```

Comment:
need to use bigInt here, the real ones will be 32 bytes

Reply:
Out of scope of this bounty. But you already forked this so i resolve conversation